### PR TITLE
SKIL-415

### PR DESCRIPTION
**Pull request by aparriaran**
TODOs Completed:
-Created Jest Tests for AdminViewTeams.
-Updated the range on the NOTE for AdminViewCourses.
-Added logic to the aria-label for EditTeam in AdminAddTeam.js.

### CODE REVIEW

File: FrontEndReact/src/View/Admin/Add/AddTeam/AdminAddTeam.js
Comment (ebanderson3) [inline FrontEndReact/src/View/Admin/Add/AddTeam/AdminAddTeam.js:1]: You should check the style formatting of this code. I imagine that this isn't intentional.

**Review by ebanderson3 — APPROVED** [review]
@sah0017 LGTM!

**Review by rtkells — APPROVED** [review]
@sah0017 LGTM!

**Comment by aparriaran** [comment]
@sah0017 I am going to merge this PR!

**Pull request merged**